packages/design-system/src/components/utilities/typographyHeadings.stories.tsx

```diff
@@ -35,17 +35,17 @@ AllHeadingsOnDark.parameters = {
 
 export const ContentClass = () => (
   <div className="ds-content">
-    <h1>Heading level 1</h1>
+    <h1>Example heading level 1</h1>
```

Comment:
Could you make use of the `heading` const on line 13?

Reply:
Good thinking 👍